File: problems/tree_magic/submissions/accepted/tree_magic.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var g int
	fmt.Scanf("%d", &g)
	result := (g-1)*int(math.Pow(float64(2), float64(g))) + 1
	fmt.Println(result)
}

File: problems/tree_magic/submissions/accepted/treeMagic.java
import java.util.Scanner;

public class treeMagic { 
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int g = Integer.parseInt(sc.nextLine());
        System.out.println( Math.round( (g-1) * (Math.pow(2, g)) + 1 ));

        sc.close();
    }
}